src/build/build_step.go

```diff
@@ -287,14 +285,13 @@ func buildTarget(tid int, state *core.BuildState, target *core.BuildTarget, runR
 		if err != nil {
 			return err
 		}
-	}
-	// The remote action will set the output directory outs here
-	if !runRemotely {
```

Comment:
just checking this is deliberate? Not sure I see how it's related...?

Reply:
Just a quick code simplification. We have an if statement above that checks runRemotely. I moved this code into the else.

Reply:
Yeah I just moved the `addOutputDirectoriesToBuildOutput(target)` into the else branch of the 
```
if runRemotely {
  ...
}
```

rather than having it in it's own if statement.